src/auxiliary/Debug.cc

```diff
@@ -199,6 +199,7 @@ void Debug::printTiles_(
                 if (multi && device > HostNum)
                     msg += ' ';
 
+                LockGuard(A.storage_->getTilesMapLock());
```

Comment:
`LockGuard guard( ... );`
Currently there's no variable name, so it creates a lock guard and immediately deletes it.